src/NUnitFramework/testdata/nunit.testdata-3.5.csproj

```diff
@@ -28,7 +28,7 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
+    <Optimize>false</Optimize>
```

Comment:
If the optimization issue was only happening in .NET 2.0, do we need to disable in all projects? Doesn't really matter with the nature of this project, just asking...